src/QuickMarcEditor/QuickMarcEditorRows/IndicatorField/IndicatorField.js

```diff
@@ -0,0 +1,18 @@
+import React, { useCallback } from 'react';
+
+import {
+  TextField,
+} from '@folio/stripes/components';
+
+export const IndicatorField = (props) => {
+  const selectContent = useCallback(({ target }) => {
```

Comment:
please cover by tests

Reply:
https://testing-library.com/docs/dom-testing-library/api-events#fireeventeventname
please use to cover onFocus